apps/server/src/modules/tldraw/entities/tldraw-drawing.entity.ts

```diff
@@ -15,6 +15,9 @@ export class TldrawDrawing {
 		if (props.action) {
 			this.action = props.action;
 		}
+		if (typeof props.part === 'number') {
```

Comment:
If it is not a number it should be undefined?

Reply:
yes, it can be undefined